docs/contributing-concept.md

```diff
@@ -0,0 +1,142 @@
+# Contributing a concept + concept exercise
+
+A [concept exercise][product-concept] aims to teach a single concept in a given language.
+It should have a clear learning goal and provide enough boilerplate to avoid the student having to learn unnecessary code.
+The associated concept has its own documentation, independent of the concept exercise.
+Concepts may have other concepts as prerequisites and be locked until the concept exercises of the prerequisites concepts are completed.
+Concepts exercises difficulty should not increase as the track progresses.
+A concept exercise should be solvable in 5 to 10 minutes by any developper familiar with the prerequisites concepts.
+
+Now that we are familiar with what a concept exercise is, let's detail how to implement one, and it's associated concept.
+For reference, the detailed [spec of a concept exercise][concept-exercise-spec] and of a [concept][concept-spec] are available in exercism [docs repository][docs-repo].
+This documents gather info from many resources to act as a convenient step-by-step guide on how to contribute a concept exercise.
+
+## 1. Identify a missing concept to implement
+
+Already implemented concepts are located in the `concepts/` directory of this repository.
```

Comment:
```suggestion
Already implemented concepts are located in the [`concepts/`](concepts/) directory of this repository.
```

Reply:
Is the link working if the file is not at the root of the repo? should it be (/concepts/) ?

Reply:
good spot, I've updated the suggestion